recipes-graphics/mesa/mesa_%.bbappend

```diff
@@ -30,9 +30,9 @@ PACKAGECONFIG_REMOVE_NXPBSP_imxgpu        = "dri"
 DRIDRIVERS_NXPBSP_imxgpu                  = ""
 #
 # For parts with GPU and DRM, use osmesa, dri, and swrast
-PACKAGECONFIG_REMOVE_NXPBSP_imxgpu_imxdrm = "gallium"
-PACKAGECONFIG_APPEND_NXPBSP_imxgpu_imxdrm = "osmesa"
-DRIDRIVERS_NXPBSP_imxgpu_imxdrm           = "swrast"
+PACKAGECONFIG_REMOVE_NXPBSP_imxgpu_imxdrm = ""
+PACKAGECONFIG_APPEND_NXPBSP_imxgpu_imxdrm = ""
+DRIDRIVERS_NXPBSP_imxgpu_imxdrm           = ""
```

Comment:
This commit is intended?

Reply:
No. You asked about whether we could trim back more. I'm still looking into it. I'll drop it for now.